### ISSUE TITLE
Allow renaming names from uploaded files

### ISSUE THREAD

**Issue by Der-Feind** [twentyhq/twenty#6317]
## Scope & Context

```
I uploaded some files now for my clients and I want to rename them inside of the CRM, because they are named otherwise in my cloud.
```

## Current behavior

```
You can just Download or Delete the items.
```

## Expected behavior
Please also add **figma screenshots or figma links** if available
 
```
It would be great, if I could easily rename Files inside of Twenty.
```

## Technical inputs

```
None.
```

**Comment by Bonapara** [comment]
Introduce a "Rename" button in the menu that turns the document label into an editable field.

![CleanShot 2024-07-18 at 18 33 54@2x](https://github.com/user-attachments/assets/5e4502f6-54a0-4c2e-8fde-6d989516109b)

https://www.figma.com/design/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=14921-60484&t=CbMpbTxQ0w9wPN1e-11

**Comment by sanidhyasin** [comment]
Hey @Bonapara, I can work on this one.

**Comment by shanthi1710** [comment]
Hey @Bonapara I would love to work on this issue

**Comment by Bonapara** [comment]
Hi @shanthi1710, just assigned @sanidhyasin, maybe you can grab another [Good first Issue](https://github.com/twentyhq/twenty/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+no%3Aassignee)?

**Comment by Bonapara** [comment]
🙏🙏 @shanthi1710

**Comment by ehconitin** [comment]
Hey @Bonapara , 
Done with this issue.
Please check [PR 6358](https://github.com/twentyhq/twenty/pull/6358).
Thanks

**Comment by Nabhag8848** [comment]
@Bonapara Can be closed, Resolved in #6358
